# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: ansgar@43-1.org-20100322095505-42dkfo2jth19fjjp
# target_branch: bzr+ssh://bzr.debian.org/bzr/apt/debian-sid/
# testament_sha1: c8369947191b33dc9b31494120f3c3789efeb5ca
# timestamp: 2010-03-22 18:55:33 +0900
# base_revision_id: jak@debian.org-20100321171146-jxyud7pq3zzwq44n
# 
# Begin patch
=== modified file 'debian/changelog'
--- debian/changelog	2010-03-21 17:11:46 +0000
+++ debian/changelog	2010-03-22 09:55:05 +0000
@@ -24,6 +24,10 @@
     - Fix the libraries name to be e.g. libapt-pkg4.9 instead of
       libapt-pkg-4.9.
 
+  [ Ansgar Burchardt ]
+  * doc/apt_preferences.5.xml:
+    - Correct mistake in example pin entry. (Closes: #514300)
+
  -- Christian Perrier <bubulle@debian.org>  Wed, 24 Feb 2010 22:13:50 +0100
 
 apt (0.7.26~exp2) experimental; urgency=low

=== modified file 'doc/apt_preferences.5.xml'
--- doc/apt_preferences.5.xml	2010-02-17 23:37:52 +0000
+++ doc/apt_preferences.5.xml	2010-03-22 09:55:05 +0000
@@ -293,7 +293,7 @@
 Pin-Priority: 999
 
 Package: *
-Pin: release unstable
+Pin: release a=unstable
 Pin-Priority: 50
 </programlisting>
 

# Begin bundle
IyBCYXphYXIgcmV2aXNpb24gYnVuZGxlIHY0CiMKQlpoOTFBWSZTWZGP1h8AAmBfgEAQWXP//3oH
TQq////wUAT4V4KCQBswlCUknpANBppobUGjRoBoAAADmAAmTAAmEwTCGAIwAJEhCMqemmaao3og
J5IwnpAZPUNADmAAmTAAmEwTCGAIwAJJCZNAjE0NU2mIZBNT1D9JNBoep6aSK9N7xVW/68DQYY9z
ixYJhkdsrnJzBU9JrljrdM/GqBceeJbQ1L3mgabb+3iFk67nEDTZmVI8Pwc5dkxt5xhBZw4v740J
WDmO1kzTAeTFuy29fJI+/CDNPqOnliO0CKtP5mksDL5pxwBqvI42+2NCCywFmAuh3jYO3xWBqlNd
R9BTdlBQChaMcqIuhbIjTIIwjLEjDPpmEKc9DHnQqhZSqELoKRKDGAUhiwAygaCgDYRJSYqoBio7
7VLxcEu9BBC7JndnxvCAQuvhK+s9qD1MCwtIWDv8UbFJDtRoNDGRGITgFa2XnhgauwB+hY324WCy
Qpq4B4WGYYk3APiYviszbg8mSNVQ4MjQoIqmf6PLwP0Am5BNk4Fdsir3DhxHUV4DGA2gjIHjEMwd
TNWtyVU2oV0EXgfJBAWBgQG1PKy/YrhYFBkNQWhroyJINqiBrOPQx2QWmkrAtKCZvYpdaOLiI8zN
JQps0QzN5qGKyv5EjWRmxKaZOKKY1yKEOeWj6B0DLCgJgy0lI8PawnVKOBCg0OuIj3mC2FdCvJF7
4vH1AMVljxh44pKSkDdkd5z7qbSpO0GNTJ5dEMGUAg9SRR6COnQve7uYNne1dmwVRLdxmQoE4Rs2
dC7MJVxJ2Sju7qi1B2Br3g66dt4aS0G35TPoMlQE2BBmV+K6TYG7gb11BFe5OfwGQMfQ4N9yIPfy
HqRkkydsm/rZOFcWnedhl+JYWTKzejj5BI8Q9e3AIbCCRTVIMuR2gxtc45HNkFC+lJiBUGOC83G0
YzGSrt0g13JLwPmYnI7iHMrPzKw9Qmc9yqG1ARWhxwvh0R22rVi3iF5YEAuBzyM0omlFTV5qAFhc
Y+GljtGC0uvY1HIduDiY/qcmqII4u8DieCVxgce/g8Mxq2R5LaOF7hVqvvW3qFcnG9a1UgrKUP19
nTqMjoWFykrF4nItWYHXwAereOABikfJKXbyPsiRnhtAZBMTIrbz+5eiN3ikwewaAd/w7olEehR6
ARY/CKPkxCluY03XQHMKBElFZlkxQI2BtA3gcA5TUFTMAN/qRmmtz4gkdZzUMRhKra4/ZuRq3I7+
pVh8ea05Ti8Fkr1Jnzssow30SlJqAYEw0p8EUKhz3BnoeHuMgxpRJgGXWg4rqPTVV8G8VKPIGD05
hJL5o+oBat5wgrDL4O8CG/UHyXDdvChTFQOK8/qHNTmmNwQiH4B6sTIK0c0ODgFY41lk1dNUAW+c
C5k+wAjEKXq90xSKDYOYzdJNqQx+w2WX1mtIhZVRAysLCkyKC6bHzlqMUeihIyEl1hmB3ZwsxAz3
kytE1JkQYNPi8GNH9BmcPer7FpWtaDu4jB6LczQM+1Qi8NGkaY8A5K12V43BvD/i7kinChISMfrD
4A==
